2016-10-14-TWIT.md

```diff
@@ -0,0 +1,32 @@
+# This Week in Tessel: Hacktoberfest, Rust update, and conferences, oh my!
+
+Hello, Tesselators!
+
+**TL;DR:**
+* [Hacktoberfest](https://hacktoberfest.digitalocean.com) is back, and we have [open issues](https://github.com/issues?utf8=✓&q=is%3Aopen+is%3Aissue+label%3AHacktoberFest+user%3Atessel+) labeled for participants!
+* We just released an update to [t2-cli](https://github.com/tessel/t2-cli/releases/tag/v0.1.0), enabling local compilation for Rust programs!
+* Jon and Kelsey will be at [JSConf Asia](https://2016.jsconf.asia) in November to speak and host a [hands-on Tessel module workshop](http://www.eventnook.com/event/devfestasiaevents2016/home)!
+
+## Hacktoberfest
+
+DigitalOcean brought [Hacktoberfest](https://hacktoberfest.digitalocean.com) back to life this month to encourage more open source contributions by sharing resources for [creating more approachable issues](https://the-pastry-box-project.net/charlotte-spencer/2015-september-16) and giving out shirts & stickers to participants. As part of our goals this year to increase contributions and encourage both inclusion and accessibility to newcomers, we've labeled [a bunch of issues](https://github.com/issues?utf8=✓&q=is%3Aopen+is%3Aissue+label%3AHacktoberFest+user%3Atessel+) for folks who want to get started!
```

Comment:
"As part of our goals this year to increase contributions and encourage both inclusion and accessibility to newcomers, we've labeled [a bunch of issues](https://github.com/issues?utf8=✓&q=is%3Aopen+is%3Aissue+label%3AHacktoberFest+user%3Atessel+) for folks who want to get started!" --> "We've labeled [a bunch of issues](https://github.com/issues?utf8=✓&q=is%3Aopen+is%3Aissue+label%3AHacktoberFest+user%3Atessel+) for folks who want to get started! Since one of our two [core goals this year](https://github.com/tessel/project#current-goals) is to increase contributions and encourage both inclusion and accessibility to newcomers, we are proud to be a part of this project."

..or something like that